src/main/scala/castalia/Routes.scala

```diff
@@ -1,17 +1,25 @@
 package castalia
 
+import akka.actor.ActorSystem
+import akka.event.{Logging, LoggingAdapter}
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
+import akka.http.scaladsl.model.StatusCodes
 import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.{RejectionHandler, Route}
 
 /**
   * Created by jml on 11/29/15.
```

Comment:
We can remove this comment since it does not give useful information. ;)
Or better, make it say something about the `Routes`